server/odc-service/src/main/java/com/oceanbase/odc/service/quartz/config/QuartzConfiguration.java

```diff
@@ -50,6 +50,8 @@ public class QuartzConfiguration {
     private Long maxConcurrentTaskCount;
 
     private final String defaultSchedulerName = "ODC-SCHEDULER";
+    private final String commonSchedulerName = "ODC-COMMON-SCHEDULER";
+    private final String APPLICATION_CONTEXT_SCHEDULER_KEY_NAME = "applicationContext";
```

Comment:
static final use UPPER variable name

Reply:
but “applicationContext” is a normal name for setApplicationContextSchedulerContextKey